test/e2e/component_test.go

```diff
@@ -10,6 +10,8 @@ import (
 	framework "github.com/operator-framework/operator-sdk/pkg/test"
 	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
 
+	devconsole "github.com/redhat-developer/devconsole-api/pkg/apis"
```

Comment:
@akashshinde We can create a separate test file for GitSource, say `test/e2e/gitsource_test.go`. 

@DhritiShikhar was working on the test for GitSource. Since the test is mostly written here. We should avoid duplicate efforts.

Reply:
@baijum As of now we don't have any controller logic in place for `GitSource`. What would be the test cases do you think we should have here ?

Reply:
This is an e2e test, which requires the operator and controller. So, we can write it here.

Reply:
Resolving this conversation since this test would go in different PR.